### ISSUE TITLE
OverflowError: Python int too large to convert to C long

### ISSUE THREAD

**Issue by Huii** [wannesm/dtaidistance#19]
First of all: Thank you very much for creating this nice piece of software! Unfortunately, I have an issue running your sample code.

Whenever I try to run the following code:

```
    series = [np.array([0, 0, 1, 2, 1, 0, 1, 0, 0], dtype=np.double),np.array([0.0, 1, 2, 0, 0, 0, 0, 0, 0, 0, 0]),np.array([0.0, 0, 1, 2, 1, 0, 0, 0])]
    ds = dtw.distance_matrix_fast(series)
    print(ds)
```
I'm greeted with this error message:
`OverflowError: Python int too large to convert to C long`

Any help would be highly appreciated.

**Comment by wannesm** [comment]
Does it also say which line is causing this? Can you also indicate your setup (Python, OS versions).

I assume that it is triggered by Python working with 64bit integers and the Cython code using 32bit. We haven't yet encountered this problem. But it seems to occur in more libraries based on a quick web search, so hopefully we can learn from them.

**Comment by Huii** [comment]
>   Does it also say which line is causing this?

Yes, it does:
```
File "Anaconda\lib\site-packages\dtaidistance-1.1.3-py3.6-win-amd64.egg\dtaidistance\dtw.py", line 451, in distance_matrix_fast
    use_c=True, use_nogil=True, show_progress=show_progress)
  File "Anaconda\lib\site-packages\dtaidistance-1.1.3-py3.6-win-amd64.egg\dtaidistance\dtw.py", line 369, in distance_matrix
    dists = dtw_c.distance_matrix_nogil_p(s, **dist_opts)
  File "dtaidistance\dtw_c.pyx", line 390, in dtaidistance.dtw_c.distance_matrix_nogil_p
  File "dtaidistance\dtw_c.pyx", line 359, in dtaidistance.dtw_c.distance_matrix_nogil
```

> Can you also indicate your setup (Python, OS versions).

I'm using Anaconda with Python 3.6 on a Windows 10 machine.

> I assume that it is triggered by Python working with 64bit integers and the Cython code using 32bit.

Is there any way I could force Cython to use 64 bit?

Initially I thought this problem might only occur in the pip package but I reinstalled dtaidistance again from source and, unfortunately, it stills persists.

**Comment by wannesm** [comment]
The C pointer type was hardcoded and is different for Windows. It is now linked to the data type defined in the stdint standard library. Fixed in commit fa547cf27d75d337f70631e01f4bea631034cb01

**Comment by Huii** [comment]
Thank you very much! It works now. :)
